tv.py

```
@@ -0,0 +1,808 @@
+      numberpart = 0
+      fewtimes = 0
+      if config.Twitch == "True":
+        try:
+          driver.find_element("xpath", "//button[@data-a-target='content-classification-gate-overlay-start-watching-button']").click()
+          time.sleep(5)
+        except Exception as e:
+          logging.error(f"Error clicking content button: {str(e)}")
+          try:
+            actions = ActionChains(driver)
+            actions.send_keys('k').perform()
+          except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)